packages/amplify-cli/src/extensions/amplify-helpers/read-json-file.js

```diff
@@ -8,6 +8,9 @@ function stripBOM(content) {
 }
 
 function readJsonFile(jsonFilePath, encoding = 'utf8') {
+  if (!fs.existsSync(jsonFilePath)) {
+    return undefined;
```

Comment:
are we sure it will not break other parts of the CLI? they depend on an object will be returned, as a prerequisite, not sure if all callers are handling undefined as the file should be there. Perhaps we need a parameter about throwIfNotExist or something....

Reply:
True I'll add a param for it to make sure it's backwards compatible